### ISSUE TITLE
Failed scheduled deliveries should email about the failure

### ISSUE THREAD

**Issue by magnew** [lightdash/lightdash#7961]
### Description

Failed scheduled deliveries should notify of the failure to send. 

In the email, we should include:

- a link to the scheduled delivery that failed (it would open the scheduled delivery modal in `edit` mode in Lightdash)
- The error message for the failure
- the time that it failed
- The title of the email should be something like `Warning: scheduled delivery failure`

This email should _**only**_ be sent to the person who created the scheduled delivery.

**Comment by PriPatel** [comment]
Yeah +1 on not messaging every user. I think whoever set up the delivery / admins should get notified. This could get annoying for board members/external users.
